app/api/v1/airline/to-airport/route.ts

```diff
@@ -0,0 +1,110 @@
+import { NextRequest, NextResponse } from "next/server"
+import { QueryResult } from "couchbase"
+
+import { getDatabase } from "@/lib/couchbase-connection"
+import { TAirline } from "@/app/models/Airline"
+
+/**
+ * @swagger
+ * /api/v1/airline/to-airport:
+ *   get:
+ *     summary: Get airlines flying to a destination airport
+ *     description: |
+ *       Get list of Airlines flying to a destination airport.
+ *    
+ *       This provides an example of using SQL++ query in Couchbase to fetch a list of documents matching the specified criteria.
+ * 
+ *       Code: [`app/api/v1/airline/to-airport/route.ts`] Method: `GET`
+ *     tags:
+ *        - Airline
+ *     parameters:
+ *       - name: destinationAirportCode
+ *         in: query
+ *         description: The ICAO or IATA code of the destination airport
```

Comment:
Can you standardize this by looking at other tutorials swagger UI, basically it also includes examples as well.
Similarly for all routes.

Reply:
have included the examples